README.md

```diff
@@ -466,6 +469,9 @@ The Visual Computer, 2021. [PDF](https://link.springer.com/article/10.1007/s0037
 - <a name="Zhang2021iccv"></a>Zhang, J., Yang, X., Fu, Y., Wei, X., Yin, B., Dong, B.,  
 *[Object Tracking by Jointly Exploiting Frame and Event Domain](https://openaccess.thecvf.com/content/ICCV2021/papers/Zhang_Object_Tracking_by_Jointly_Exploiting_Frame_and_Event_Domain_ICCV_2021_paper.pdf)*,  
 IEEE Int. Conf. Computer Vision (ICCV), 2021. [Project](https://zhangjiqing.com/publication/iccv21_fe108_tracking/), [PDF](https://arxiv.org/abs/2109.09052), [code](https://github.com/Jee-King/ICCV2021_Event_Frame_Tracking), [dataset](https://zhangjiqing.com/dataset/).
+- <a name="dietsche21iros"></a>A. Dietsche, G. Cioffi, J. Hidalgo-Carrio, D. Scaramuzza,  
+*[Powerline Tracking with Event Cameras](https://arxiv.org/abs/2108.00515)*,  
```

Comment:
If there is an official publication venue (IROS, etc.), we typically prefer to have the doi link in the title.
It applies to all entries below, too.